src/IDatabaseExtensions.TempTable.cs

```diff
@@ -0,0 +1,265 @@
+﻿using Kros.Data.BulkActions;
+using Kros.KORM.Data;
+using Kros.KORM.Extensions;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Kros.KORM
+{
+
+    public partial class IDatabaseExtensions
```

Comment:
What will these extension methods do, if connected database is not SQL server database?